capabilityquerier/pkg/capabilityquerier/capabilityquerier_test.go

```diff
@@ -47,7 +49,7 @@ func Test_GetAndSendCapabilityStatement(t *testing.T) {
 	// create the expected result
 	expectedCapStat, err := capabilityStatement()
 	th.Assert(t, err == nil, err)
-	expectedMimeType := []string{fhir2LessJSONMIMEType, fhir3PlusJSONMIMEType}
+	expectedMimeType := []string{fhir2LessJSONMIMEType, fhir3PlusJSONMIMEType, fhir2LessJSONMIMEType, fhir3PlusJSONMIMEType}
```

Comment:
why are mime type entries being duplicated here?

Reply:
The Mime types were added twice because the part of the code that gets this in requestCapabilityStatement function would be called twice. I changed it to only add the mime types for the metadata call only now